lib/parse.js

```diff
@@ -5,7 +5,7 @@ export const number = function(string) {
   if (typeof string === 'number') {
     return string;
   }
-  if (string === '') {
+  if (string === '' || string === undefined) {
```

Comment:
Is it possible this will over-coerce? That is, hide some errors we’d want to know about?

Reply:
Good point. 

In my case I'm getting `undefined` due to the way the scraped data is structured, so I've handled that within the scraper. 

I've changed parse.number so that it explicitly checks for `undefined` and throws a more informative error, rather than waiting until it tries to do `.replace` on an undefined value.